### PR TITLE
Add support for stopTimeout in container definition

### PR DESCRIPTION
**Pull request by rajmaniar**

### CODE REVIEW

**Comment by rajmaniar** [comment]
Done!
Thanks for the reminder...you might not be the first person to remind me of this ;-)

File: main.tf [hunk 1]
```diff
@@ -19,6 +19,7 @@ locals {
     volumesFrom            = "${var.volumes_from}"
     user                   = "${var.user}"
     dependsOn              = "${var.depends_on}"
+    stopTimeout            = "stop_timeout"
```
Comment (aknysh) [inline main.tf]:
```suggestion
    stopTimeout            = "stop_timeout_sentinel_value"
```

for consistency with other fields

File: outputs.tf [hunk 2]
```diff
@@ -10,15 +10,17 @@ locals {
...
+  encoded_stop_timeout          = "${var.stop_timeout > 0 ? var.stop_timeout : "null"}"
   encoded_container_definition  = "${replace(replace(replace(jsonencode(local.container_definition), "/(\\[\\]|\\[\"\"\\]|\"\"|{})/", "null"), "/\"(true|false)\"/", "$1"), "/\"(-?[0-9]+\\.?[0-9]*)\"/", "$1")}"
 
   json_with_environment        = "${replace(local.encoded_container_definition, "/\"environment_sentinel_value\"/", local.encoded_environment_variables)}"
   json_with_secrets            = "${replace(local.json_with_environment, "/\"secrets_sentinel_value\"/", local.encoded_secrets)}"
   json_with_cpu                = "${replace(local.json_with_secrets, "/\"cpu_sentinel_value\"/", local.encoded_cpu)}"
   json_with_memory             = "${replace(local.json_with_cpu, "/\"memory_sentinel_value\"/", local.encoded_memory)}"
   json_with_memory_reservation = "${replace(local.json_with_memory, "/\"memory_reservation_sentinel_value\"/", local.encoded_memory_reservation)}"
+  json_with_stop_timeout       = "${replace(local.json_with_memory_reservation, "/\"stop_timeout\"/", local.encoded_stop_timeout)}"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  json_with_stop_timeout       = "${replace(local.json_with_memory_reservation, "/\"stop_timeout_sentinel_value\"/", local.encoded_stop_timeout)}"
```

File: variables.tf [hunk 3]
```diff
@@ -154,3 +154,8 @@ variable "depends_on" {
   description = "The dependencies defined for container startup and shutdown. A container can contain multiple dependencies. When a dependency is defined for container startup, for container shutdown it is reversed"
   default     = []
 }
+
+variable "stop_timeout" {
+  description = "Timeout in seconds between sending container SIGTERM and SIGKILL"
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  description = "Timeout in seconds between sending SIGTERM and SIGKILL to container"
```

**Comment by rajmaniar** [comment]
> thanks @rajmaniar
> one more thing:
> please rebuild README by executing these commands:
> 
> ```
> make init
> make readme/deps
> make readme
> ```
> 
> It will add the new variables and outputs to `README.md` automatically.
> 
> In general, any changes to README should be made in `README.yaml` (not in this case), and after that executing the commands above will rebuild `README.yaml` into `README.md` and add all new variables and outputs to `README.md`
> 
> thanks

Got it, done!

**Pull request merged**